app/Image/SizeVariantDefaultFactory.php

```diff
@@ -358,8 +381,11 @@ protected function getMaxDimensions(int $sizeVariant): array
 	 * configurable and are always enabled (e.g. a thumb).
 	 * Hence, it is save to call this function for all size variants.
```

Comment:
```suggestion
	 * Hence, it is safe to call this function for all size variants.
```
Not your edit but it was visible in the context diff...